init.lua

```diff
@@ -154,9 +213,16 @@ function SkyRocket:handleCancel()
 
     if self:isResizing() then
       self:resizeWindowToCanvas()
-    else
+    end
+    if self:isMoving() then
```

Comment:
Why did you separate these out into separate `if` statements to evaluate?

I'd think they'd be mutually exclusive, so a:

```lua
if isMoving
else if isFancy
else if isResizing
end
```

would make more sense to me.

Reply:
It was my first day programming lua, I changed all the mutually exclusive ifs to the format you suggest.

I miss switch statements tho